### PR TITLE
feat(DynamicWidgets): release as stable

### PR DESCRIPTION
**Pull request by Haroenv**
This widget is fully available now, and thus no longer experimental.

DX-2420

### CODE REVIEW

**Review by eunjae-lee — APPROVED** [review]
Let's go!

**Review by dhayab — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -122,8 +122,6 @@
     "react-autosuggest": "9.4.3",
     "react-dom": "16.14.0",
     "react-error-boundary": "3.1.3",
-    "react-instantsearch-dom": "6.12.0",
-    "react-instantsearch-dom-maps": "6.12.0",
```
Comment (francoischalifour) [inline package.json:126]: Why this change?

Comment (Haroenv) [reply package.json:126]: they didn't seem needed, but it caused the stories to import from node_modules instead of the local packages @francoischalifour

**Review by francoischalifour — APPROVED** [review]
